nilearn/decomposition/canica.py

```diff
@@ -106,30 +109,31 @@ class CanICA(MultiPCA, CacheMixin):
       datasets", IEEE ISBI 2010, p. 1177
     """
 
-    def __init__(self, mask=None, n_components=20,
-                 smoothing_fwhm=6, do_cca=True,
-                 threshold='auto', n_init=10,
-                 standardize=True,
-                 random_state=0,
-                 target_affine=None, target_shape=None,
+    def __init__(self, n_init=1, threshold='auto', n_components=20,
```

Comment:
You changed n_init default from 10 to 1, is that on purpose ?

Not sure why you reordered the parameters, but that make things like this a lot more likely to pass through the cracks ...